src/locales/en.json

```diff
@@ -424,6 +425,7 @@
         "month": "monthly",
         "no_match": "No rates were found",
         "no_range_set": "No range set",
+        "node currency rate": "Node price rate tier {{index}} ({{ unit }})",
```

Comment:
Can we please format keys using the existing naming convention (e.g., "node_currency_rate")?

Also, I don't see "node currency rate" and "cluster currency rate" used anywhere. Is this the only file needing to be updated?

Reply:
Yes it does come from the API :+1: - otherwise I wouldn't use such a weird naming convention.

Reply:
Didn't realize the UI was localizing API strings. 

That will be a problem because if the API changes, it can easily break the UI.

Reply:
FYI, I created issue https://github.com/project-koku/koku-ui/issues/1537.